database/seeds/UsersTableSeeder.php

```diff
@@ -19,6 +19,7 @@ public function run() {
         
         factory(App\User::class, 10)->create()->each(function ($user) {
             factory(App\MacAddress::class, $user->id % 5)->create(['user_id' => $user->id]);
+            factory(App\PrintJob::class, $user->id % 5)->create(['user_id' => $user->id]);
```

Comment:
I'm not sure where mention this.
The seeded print jobs in the history table all has almost the same date.
Probably different dates would improve the *realness* of the records.